File: api/hospital-system/src/main/java/com/api/hospitalsystem/model/heathPlan/HeathPlan.java
package com.api.hospitalsystem.model.heathPlan;

public enum HeathPlan {

    Amil("Amil"),
    BradescoSaude("Bradesco Saúde"),
    Hapvida("Hapvida"),
    InterMedica("Intermédica"),
    NotredameIntermedica("NotreDame Intermédica"),
    SulAmerica("SulAmérica"),
    Unimed("Unimed"),
    GoldenCross("Golden Cross"),
    PortoSeguro("Porto Seguro"),
    SaoCristovao("São Cristóvão"),
    None("Nenhum");

    private final String description;

    HeathPlan(String description) {
        this.description = description;
    }

    public String getValue() {
        return description;
    }

    @Override
    public String toString() {
        return description;
    }

}

File: app/hospital-system/src/app/module/models/src/heath-plan/heath-plan.ts
import { describe } from '@module/utils/functions/enum';

export enum HeathPlan {
  Amil = 0,
  BradescoSaude = 1,
  Hapvida = 2,
  InterMedica = 3,
  NotredameIntermedica = 4,
  SulAmerica = 5,
  Unimed = 6,
  GoldenCross = 7,
  PortoSeguro = 8,
  SaoCristovao = 9,
  None = 10,
}

describe(HeathPlan, {
  Amil: 'Amil',
  BradescoSaude: 'Bradesco Saúde',
  Hapvida: 'Hapvida',
  InterMedica: 'Intermédica',
  NotredameIntermedica: 'NotreDame Intermédica',
  SulAmerica: 'Sul América',
  Unimed: 'Unimed',
  GoldenCross: 'Golden Cross',
  PortoSeguro: 'Porto Segur',
  SaoCristovao: 'São Cristóvão',
  None: 'Nenhum',
});
